File: app/src/main/java/com/chirag/covid19india/ApplicationClass.java
package com.chirag.covid19india;

import android.content.Context;
import android.os.StrictMode;

import androidx.multidex.MultiDexApplication;


public class ApplicationClass extends MultiDexApplication {

    private static ApplicationClass context;

    @Override
    public void onCreate() {
        super.onCreate();
        context = this;
        StrictMode.VmPolicy.Builder builder = new StrictMode.VmPolicy.Builder();
        StrictMode.setVmPolicy(builder.build());
    }

    public static Context getAppContext() {
        return context;
    }

}

File: app/src/main/java/com/chirag/covid19india/ApplicationClass.kt
package com.chirag.covid19india

import android.os.StrictMode
import android.os.StrictMode.VmPolicy
import androidx.multidex.MultiDexApplication

class ApplicationClass : MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
        val builder = VmPolicy.Builder()
        StrictMode.setVmPolicy(builder.build())
    }
}